content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -1,18 +1,115 @@
-<admon>
+# Run Experiments
+
+Using your own cloud infrastructure, you can train your model on the cloud and
+run experiments with different hyperparameters or different datasets.
+
+> Due to access restrictions, you cannot run experiments on the demo projects
+> that are provided to you by default. Once you connect to your ML project
+> repositories, you can follow the instructions given below to run experiments
+> directly from Iterative Studio.
+
+## Cloud experiments
 
-**We have renamed Views to Projects in Iterative Studio.**
+<admon>
 
-Accordingly, _Views dashboard_ is now called _Projects dashboard_; _View
-settings_ are now called _Project settings_; and so on.
+Cloud experiments are in alpha release and are subject to change.
 
 </admon>
 
-# Run Experiments
+Run your experiments on your own cloud compute instances in AWS, Azure, and GCP.
+To get started, you need:
+
+1. Credentials for the cloud provider.
+2. A [DVC experiment pipeline]. The default [script] runs a DVC pipeline,
+   although advanced users can run without a pipeline by modifying the script.
+3. A `requirements.txt` file listing Python packages required by the project.
+   The default [script] installs all requirements from `requirements.txt`,
+   although advanced users can specify requirements differently by modifying the
+   script.
+
+[script]: #submit-an-experiment
+
+### Setup Credentials
+
+To enable cloud experiments, first add credentials (or use credentials from a
+[data remote]). To add credentials, click on your user icon in the top right
+corner and go to your [Profile]. Navigate to [Cloud Credentials] and click
+`Add credentials`. Select either AWS, Azure, or GCP as the provider. Depending
+on the provider, you will be asked for more details. Finally, click
+`Save credentials`.
+
+![](https://static.iterative.ai/img/studio/s3_remote_settings_v2.png)
+
+The credentials must have the required permissions. See details and examples for
+each provider below:
+
+<details>
+
+#### AWS
```

Comment:
TODO: Add details and examples for each cloud provider

Reply:
IMO, we should include all details about credentials in a single place, so that users don't have to read different pages to learn about credentials for projects and cloud exps. So let's update the sections `/doc/studio/user-guide/account-management#cloud-credentials` and  `/doc/studio/user-guide/projects-and-experiments/configure-a-project#data-remotes--cloud-storage-credentials`. These sections already have most of the info, and only need to be updated to make them generic, so that they don't relate only to data remotes. I'll update those sections. Then, we can link to these sections from the cloud exps docs page.

Reply:
Making those other sections generic and linking there for how to add credentials makes sense. 

I think we will still need experiment-specific instructions and examples because the permissions required to run experiments differ from those needed for data remotes.